services/app/src/quests/learning_to_adventure.xml

```diff
@@ -47,15 +47,27 @@
         <p>You consider your options:</p>
         <choice text="Try to intimidate the fairies" if="intimidated == 0">
           <roleplay icon="fae" title="Intimidation Tactics" data-line="60">
-            <p>{{intimidated = 1}}</p>
             <p>You march up to them, a hand on the hilt of your newly acquired sword.</p>
             <p>"As guardsmen of the town watch, we demand you stop disturbing the peace!"</p>
-            <p>...</p>
-            <p>They ignore you. You try again. "STOP!" you yell as loud as you can.</p>
-            <p>Nothing. You'll have to try a different strategy.</p>
+            <choice text="Rolled above 10">
+                <p>{{intimidated = 1}}</p>
+                <p>You successfully intimidate the fairies</p>
```

Comment:
Perhaps this should go to the combat directly? Seems weird to have it say that intimidation succeeded, and then go back to the "try another option" screen

Reply:
I modified this outcome to flow directly into the subue/force scene as a transition into the arrest scene which then flows into combat.